### ISSUE TITLE
Not working on chrome version 75 on android

### ISSUE THREAD

**Issue by igortxo14** [brinley/jSignature#169]
jsignature is working perfect on android mobile chrome v74 or lower, and firefox.

Can't sign on android mobile chrome v75

**Comment by nvacheishvili** [comment]
I am having the same issue. Has anyone found a solution? I am thinking that it might be an issue with the touchstart events

**Comment by 4bdesign** [comment]
Yeah I'm having the same issue too. I have found it works the first time the page is loaded but if it is refreshed or re-visited it doesn't work.

Would love to hear if anyone has found a solution.

**Comment by igortxo14** [comment]
i found a solution for me.
I've added some css to the code.

On the canvas css style:

       -webkit-user-drag: none;
        user-select: none;     

On the div parent of the canvas css:

            touch-action: none;
            -webkit-touch-callout: none;
            /* iOS Safari */
            -webkit-user-select: none;
            /* Chrome/Safari/Opera */
            -khtml-user-select: none;
            /* Konqueror */
            -moz-user-select: none;
            /* Firefox */
            -ms-user-select: none;
            /* Internet Explorer/Edge */
            user-select: none;
             overflow: hidden;
             -webkit-transition: none !important;
            -moz-transition: none !important;
            -o-transition: none !important;
            -ms-transition: none !important;
            transition: none !important;

**Comment by 4bdesign** [comment]
Hmmm, I couldn't get that working for me, didn't seem to make any difference. Thanks for sharing though :)

**Comment by nvacheishvili** [comment]
Hey, thanks for sharing your solution! It helped me!  @igortxo14

**Comment by nvacheishvili** [comment]
Unfortunately, this fix is not a permanent solution as I have discovered. After several tests, the solution started to fail.  The issue is due to chrome's update, you can follow more information here: 
https://bugs.chromium.org/p/chromium/issues/detail?id=971937&q=canvas&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified 
If anyone discovers a permanent solution, please share 🥇 💯

**Comment by caot** [comment]
Can anyone test the way mentioned at https://github.com/brinley/jSignature/issues/135 ?
